Sources/Datadog/Logging/RemoteLogger.swift

```diff
@@ -37,18 +37,18 @@ internal final class RemoteLogger: LoggerProtocol {
     /// Date provider for logs.
     private let dateProvider: DateProvider
     /// Integration with RUM. It is used to correlate Logs with RUM events by injecting RUM context to `LogEvent`.
-    /// Can be `nil` if the integration is disabled for this logger or if RUM feature is disabled.
-    internal let rumContextIntegration: LoggingWithRUMContextIntegration?
+    /// Can be `false` if the integration is disabled for this logger or if RUM feature is disabled.
+    internal let rumContextIntegration: Bool
     /// Integration with Tracing. It is used to correlate Logs with Spans by injecting `Span` context to `LogEvent`.
-    /// Can be `nil` if the integration is disabled for this logger or if Tracing feature is disabled.
-    internal let activeSpanIntegration: LoggingWithActiveSpanIntegration?
+    /// Can be `false` if the integration is disabled for this logger or if Tracing feature is disabled.
+    internal let activeSpanIntegration: Bool
```

Comment:
Feels like the _"or if (...) feature is disabled_" part of these comments are not true anymore, no?